index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Alecxis</h1>
+    <h2>About</h2>
+    <p>I am a creative Tech/Entrepreneur from Philadelhia, now I am worldwide.</p>
+    <h2>Experience</h2>
+    <li>Portfolio Website, Product Website, Kitchen Designer, Product Designer</li>
+    <h2>Connect</h2>
+    <ul>ylleksixcela@aol.com</ul>
```

Comment:
Non blocking again but I'll give you a challenge. When users click on this entry, they could navigate to your socials or get a popup to send you an email if you use `<a>` tags. Read more about them here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a